images/macos/provision/core/xamarin.sh

```diff
@@ -64,6 +64,10 @@ createBundleLink $CURRENT_SDK_SYMLINK "Current"
 # Fix nuget in some mono versions because of known bugs
 #
 
+if is_BigSur || is_Catalina; then
```

Comment:
let's leave a comment that we need this because of the issue https://github.com/mono/mono/issues/21180

Reply:
Thank you. Done.